pkgs/development/libraries/libgtop/default.nix

```diff
@@ -26,12 +27,23 @@ stdenv.mkDerivation rec {
     perl
     gettext
     gobject-introspection
+
+    # required due to patching the Makefile.am file
+    autoreconfHook
   ];
 
   propagatedBuildInputs = [
     glib
   ];
 
+  postPatch = ''
+    substituteInPlace src/daemon/Makefile.am \
+      --replace "\`uname -s\`" "DUMMY_KERNEL_NAME" \
+      --replace "\`uname -r\`" "DUMMY_KERNEL_RELEASE" \
+      --replace "\`uname -v\`" "DUMMY_KERNEL_VERSION" \
+      --replace "\`uname -m\`" "DUMMY_KERNEL_MACHINE"
```

Comment:
i added deterministic-uname in https://github.com/NixOS/nixpkgs/pull/210102 so this can be removed